src/app/settings/settings.component.html

```diff
@@ -7,10 +7,12 @@
         <div id="sidebar-wrapper">
             <h1>Lookups</h1>
             <ul class="settings-links">
+                <li><a [class.hidden]="loggedInRole !== 'Administrator'" [routerLink]="['agencies']" routerLinkActive="active">Agencies</a></li>
```

Comment:
Same as above, we can actually remove these checks for the loggedinrole == admin since the settings pages are only accessible by